gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
-#Mon Aug 21 18:13:41 CEST 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-9.0.0-bin.zip
```

Comment:
My distribution (NixOS) distributes gradle `8.14.3` and I'd prefer if we use that instead of gradle `9`. The reason is that I'd wish to reproduce the wrapper update.

Reply:
[This PR](https://github.com/NixOS/nixpkgs/pull/421047) seems to suggest that NixOS recently updated to Gradle 9, so this should be fixed with a system update

Reply:
These take a long time to get into the unstable branch, and even more time to get into my system. I'd prefer to avoid compatibility issues for contributors.

Reply:
I have now downgraded Gradle by invoking the wrapper again.
I had to call `gradlew wrapper` twice until it actually downgraded the jar, but it should now be using consistent versions for the jar, properties, and scripts.